providers/shared/components/component.ftl

```diff
@@ -54,6 +54,8 @@
     type=DB_COMPONENT_TYPE
 /]
 
+[#assign DDB_COMPONENT_TYPE = "ddb" ]
```

Comment:
ddb doesn't really mean anything as a Component name. Why not docdb?

Reply:
Done